
From: Paul Jackson <pj@sgi.com>

Now that the emulation of the obsolete cpumask macros is no longer needed,
remove it from cpumask.h

Signed-off-by: Paul Jackson <pj@sgi.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/linux/cpumask.h |   12 ------------
 1 files changed, 12 deletions(-)

diff -puN include/linux/cpumask.h~cpumask-9-10-remove-no-longer-used-obsolete-macro-emulation include/linux/cpumask.h
--- 25/include/linux/cpumask.h~cpumask-9-10-remove-no-longer-used-obsolete-macro-emulation	Thu Jun  3 17:11:46 2004
+++ 25-akpm/include/linux/cpumask.h	Thu Jun  3 17:11:46 2004
@@ -364,16 +364,4 @@ extern cpumask_t cpu_present_map;
 #define for_each_online_cpu(cpu)  for_each_cpu_mask((cpu), cpu_online_map)
 #define for_each_present_cpu(cpu) for_each_cpu_mask((cpu), cpu_present_map)
 
-/* Begin obsolete cpumask operator emulation */
-#define cpu_isset_const(a,b) cpu_isset(a,b)
-#define cpumask_const_t cpumask_t
-#define cpus_coerce(m) (cpus_addr(m)[0])
-#define cpus_coerce_const cpus_coerce
-#define cpus_promote(x) ({ cpumask_t m; m.bits[0] = x; m; })
-#define cpus_weight_const cpus_weight
-#define first_cpu_const first_cpu
-#define mk_cpumask_const(x) x
-#define next_cpu_const next_cpu
-/* End of obsolete cpumask operator emulation */
-
 #endif /* __LINUX_CPUMASK_H */
_
